src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/AbstractEntityAIGuard.java

```diff
@@ -88,6 +91,26 @@ public AbstractEntityAIGuard(@NotNull final J job)
         );
         worker.getCitizenExperienceHandler().setSkillModifier(2 * worker.getCitizenData().getStrength() + worker.getCitizenData().getIntelligence());
         worker.setCanPickUpLoot(true);
+        
+        itemsNeeded.add(new GuardItemsNeeded(EntityEquipmentSlot.FEET, Items.LEATHER_BOOTS, 1,2,3));
```

Comment:
I think we can add this dynamically in the method getItemsNeeded depending on the level of the building?